sdk/storage/Azure.Storage.Blobs/src/BlobClient.cs

```diff
@@ -1163,6 +1067,10 @@ string GetNewBase64BlockId(long blockOrdinal)
         }
         #endregion Upload
 
+        // NOTE: TransformContent is no longer called by the new implementation
```

Comment:
??

Reply:
This is used by the upcoming encryption work so I didn't want to completely remove it.  It'll need to be reintegrated in the new implementation.

Reply:
Linked to your encryption bug.